tools/yaml_to_json.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
+# -*- coding: UTF-8 -*-
+
+# Copyright 2016 Workiva Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# system imports
+import argparse
+import json
+
+# library imports
+
+# application imports
+from aws_lambda_fsm.config import get_current_configuration
+from aws_lambda_fsm.constants import CONFIG
+
+# setup the command line args
+parser = argparse.ArgumentParser(description='Turns a fsm.yaml file into a AWS Step Function JSON definition.')
```

Comment:
The filename and description here seems to be misleading — it suggestions yaml -> json formation translation.